<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yang</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/lx.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
background-image: url(:/new/prefix1/blank.JPG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
background-image: url(:/new/prefix1/blank.JPG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="myBtn" name="btn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-image: url(:/new/prefix2/chuyin.png);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
background-image: url(:/new/prefix1/blank.JPG);
font: 75 12pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>步数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
background-image: url(:/new/prefix1/blank.JPG);
font: 75 12pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
font: 75 12pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>参考图片</string>
    </property>
   </widget>
   <widget class="myBtn" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-image: url(:/new/prefix2/chuyin.png);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="myBtn" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>382</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-image: url(:/new/prefix2/chuyin.png);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>重来开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
background-image: url(:/new/prefix1/blank.JPG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
background-image: url(:/new/prefix1/blank.JPG);
font: 75 12pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>分数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix2/chuyin.png);
font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>拼图游戏</string>
    </property>
   </widget>
   <zorder>score_label</zorder>
   <zorder>label_6</zorder>
   <zorder>label_4</zorder>
   <zorder>time_label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>btn</zorder>
   <zorder>label</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_7</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myBtn</class>
   <extends>QPushButton</extends>
   <header location="global">mybtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
